SYCL/ESIMD/regression/dgetrf_8x8_aspect-fp64.cpp

```diff
@@ -0,0 +1,18 @@
+//==------- dgetrf_8x8_aspect-fp64.cpp  - DPC++ ESIMD on-device test -------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// Enable FP64 part of <dgetrf_8x8.cpp>. To be removed once DPC++
+// supports optional device features and the code could be enabled
+// unconditionally without causing failures in speculative compilation
+// of the kernels.
```

Comment:
I think I've mentioned that before, maybe for another test. This is incorrect as doubles aren't filtered out in the included sources (to the best of my understanding). Instead, we have a selection between float/double and they cannot co-exist at the same time.

Reply:
Fixed in `dgetrf.cpp` by using function template.